flask_app/models/project.py

```diff
@@ -47,3 +47,46 @@ def get_all_projects(cls):
 
                 all_projects.append(project_obj)
             return all_projects
+        
+    @classmethod
+    def get_one_project(cls,data):
+        query = "SELECT * FROM project JOIN users ON project.user_id = users.id WHERE project.id = %(id)s;"
+        results = connectToMySQL(cls.db).query_db(query,data)
+        print(results)
+        
+        if len(results) == 0:    
+            return None
+        else: 
+            project_dictionary = results[0]
+            project_obj = cls(results[0])
+            user_dictionary = {
+                    "id" : project_dictionary['users.id'],
+                    "first_name" : project_dictionary['first_name'],
+                    "last_name" : project_dictionary['last_name'],
+                    "email" : project_dictionary['email'],
+                    "year" : project_dictionary ['year'],
+                    "password" : project_dictionary['password'],
+                    "created_at" : project_dictionary['users.created_at'],
+                    "updated_at" : project_dictionary['users.updated_at'],
+                }
+
+            project_obj = User(user_dictionary)
+
+            project_obj.user = user_obj
+            return project_obj
+        
+    @staticmethod
+    def validate_project(form_data):
+        is_valid = True
+        print(form_data)
+        return is_valid   # Please let me know desired validations so I can enter them
```

Comment:
@aacole9 , great question. I will bring this up in the discord server tomorrow morning!